src/NuGet.Services.EndToEnd/NuGet.Services.EndToEnd.csproj

```diff
@@ -95,10 +95,7 @@
     <Content Include="Support\NuGetExe\nuget.4.1.0.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="Support\NuGetExe\nuget.4.3.0-preview1-4044.exe">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Support\NuGetExe\nuget.4.6.1.exe">
+    <Content Include="Support\NuGetExe\nuget.4.7.0.exe">
```

Comment:
Out of curiosity, why are we keeping `4.1.0` but replacing the other versions?

Reply:
4.1.0 is still used by tests due to a behavior change in the client.